### PR TITLE
Clothings lockers stuff and other goodies

### PR DESCRIPTION
**Pull request by ghost**
- Added medical wardrobe
- Added EMT (first responder) jackets (labcoats) (port from bay, someone can make better sprites if they want)
- Reorganized things in medical storage
  ss 2014-06-17 at 02 55 33
- Nerfed jacket and camo pants spawn rate in sec lockers (there's two sec wardrobes now in the brig, no need for so many free armors that aren't even behind a lock!)

~~\- Gave medical doctors pepperspray to defend their purity from greyshirts raiding medical for syringe guns and spray bottles~~
- Added folded bodybag to medical doctors on spawn, STOP DRAGGING DEAD BODIES AROUND ON THE FLOOR YOU FUCKERS
- Removed satchels from the tox/chem/viro lockers on the map, added them to the lockers in the actual code

This will hopefully move MDs more to a first responder and medic role like on bay.
- Added new cool potted plants around the map! Wait and see~
  (hint, cherry blossoms for stutterwhores)
- Fixed the broken chairs in the cap's office

### CODE REVIEW

**Comment by ghost** [comment]
ss 2014-06-17 at 03 46 57

cmo has a CMC now

**Comment by Metacide** [comment]
All of these changes sound great, though I have slight reservations about the pepperspray. Live testing should be interesting, though, and as you say it can be changed depending on how things go. Bodybags on spawn is a great idea. Satchels in code is handy.

More plants sounds great - loving the CMO one. Always wished there was more variety in the sprites.

All in all looks great. :+1:

**Comment by ghost** [comment]
yeah they are but you have to right click them and use 'load from icon states' or whatever because they're not different plants but just icon swaps for the same object

also i'm not sure about the pepperspray thing, it seems like it'd be something that would be fun to try out that is all

**Comment by ghost** [comment]
the idea was that since they have access to syringe guns, why not pepperspray? but I guess you have a point.

also it's a free mini spray bottle for mass lubing and that might be a problem.

**Comment by Atskadan** [comment]
you can strikeout the pepperspray change in the description so people dont get confused

**Comment by ghost** [comment]
i fixed normal traitors being able to buy semi auto shotgun ammo, small oversight

**Comment by Metacide** [comment]
Could you give me an @Metacide when this goes through? I'd like to do some tinkering with the map after that, if that's alright. Should only take a day or two, plan to add a few nice things in space and a few other tweaks I meant to get round to a while back. Will also update the wiki page with pictures and all the updates from recent times.

**Comment by ghost** [comment]
Yeah that's fine, this should be my last update to the map since you're around again.

File: code/modules/clothing/shoes/colour.dm [hunk 1]
```diff
@@ -60,19 +64,20 @@
...
 	name = "white shoes"
+	desc = "A pair of sterile white shoes."
 	icon_state = "white"
 	permeability_coefficient = 0.01
 	item_color = "white"
 
 /obj/item/clothing/shoes/sneakers/rainbow
 	name = "rainbow shoes"
-	desc = "Very gay shoes."
+	desc = "Very happy shoes!"
```
Comment (ghost) [inline code/modules/clothing/shoes/colour.dm:80]: fucking coders are so mean

**Comment by ghost** [comment]
updated more locker shit, look at the commit too lazy to changelog

also merge this metacide needs to get his fingers back into mapping

File: code/game/objects/structures/crates_lockers/closets/secure/medical.dm [hunk 2]
```diff
@@ -69,32 +69,16 @@
...
-		switch(pick("blue", "green", "purple"))
-			if ("blue")
-				new /obj/item/clothing/under/rank/medical/blue(src)
-			if ("green")
-				new /obj/item/clothing/under/rank/medical/green(src)
-			if ("purple")
-				new /obj/item/clothing/under/rank/medical/purple(src)
-		new /obj/item/clothing/under/rank/medical(src)
-		new /obj/item/clothing/suit/labcoat(src)
-		new /obj/item/clothing/shoes/sneakers/white(src)
```
Comment (Atskadan) [inline code/game/objects/structures/crates_lockers/closets/secure/medical.dm:90]: why remove this stuff? im sure quite a few people like to wear these clothes
edit im a fucking retard nevermind

**Comment by ghost** [comment]
- Added autoinjectors (medipens) to replace all the stupid fucking 15u inaprovaline syringes around the station/in medikits and shit because fuck waiting to inject someone with inaprov, hopefully people will actually use these to stabilize people in crit from now on
- Hyposprays fit in medical belts now (medipens by extension)
- Added EMT (first-responder) jackets for medical doctors in the wardrobe locker, with new sprites by MrSnapwalk
  ss 2014-06-20 at 04 29 16
- Added steel-toed boots for first-responders as well, they're similar to jackboots but shorter, and put a couple pairs in maint.

That's it I think. Merge plz.

**Comment by RemieRichards** [comment]
Merging dis. pinging @Metacide ~

**Comment by Metacide** [comment]
New first-responder jacket sprites are great. :+1: 

Will take a look at the map later today, thanks for the ping, RemieRichards.

**Comment by Atskadan** [comment]
on bay medipens are one use instant injectors
im guessing its the same

**Pull request merged**